apps/src/music/views/MusicView.jsx

```diff
@@ -324,6 +327,13 @@ class UnconnectedMusicView extends React.Component {
     );
   };
 
+  onVideoClosed = () => {
+    this.setState({showingVideo: false});
+
+    const videoDuration = (Date.now() - this.state.sessionStartTime) / 1000;
+    this.analyticsReporter.onVideoClosed('initial-modal-0', videoDuration);
```

Comment:
I had considered that, and just opted to track separately so that the Analytics Reporter didn't have to contain business logic specific to this first video popup.  That way, if we add additional videos shown at different points, the reporter itself wouldn't have to know about such changes.  Easily convinced otherwise though.

Reply:
Gotcha, maybe in that case it makes more sense to call the reporter directly inside `Video` then? It can start the timer when it mounts and can call the AnalyticsReporter on close. That way if we have multiple, each Video component handles its own timer and reporting.

Reply:
Good idea.  Updated.